README.md

```diff
@@ -83,24 +82,27 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://fascalfocus1.onrender.com/)
```

Comment:
- [ ] The application should be deployed using Heroku, Netlify or GitHub Pages.
- [ ] The project should be deployed properly, and the home and detail page should work as expected. right now the page is not loading.

Reply:
- This is a rails app. render is the recommended option by microverse,
- Secondly, please use a better browser. its working perfectly on chrome

# View:
![image](https://user-images.githubusercontent.com/50990925/219867072-865bdbad-643f-4e4e-bd2a-aabf2a8b845c.png)